gcsa/google_calendar.py

```diff
@@ -371,7 +371,7 @@ def get_events(
             single_events=False,
             query=None,
             **kwargs
-    ):
+    ) -> Generator[gcsa.event.Event, None, None]:
```

Comment:
The style check fails as `gcsa.event.Event` is not imported here.

Also, I'd avoid using `Generator` type to avoid confusion with `None`s. I'd use `Iterable[Event]` (without verbose `gcsa.event.` part).

Reply:
And as we are at it, we should also add a type annotation to the `get_instances` method (same thing).

Reply:
done